src/store/ChartState.js

```diff
@@ -258,51 +258,50 @@ class ChartState {
             this.rootNode = this.mainStore.chart.rootNode;
         }
         this.rootNode.querySelector('.chartContainer').style.backgroundColor = Theme[`${theme}_chart_bg`];
-        // change chart colors to grey if the current market is closed and it is not a static chart
+        // reduce opacity to 32% if the current market is closed and it is not a static chart
         if (isChartClosed && !this.isStaticChart) {
-            const closedChartColor = 'rgba(129, 133, 152, 0.35)';
-            this.stxx.setStyle('stx_mountain_chart', 'borderTopColor', closedChartColor);
+            this.stxx.setStyle('stx_mountain_chart', 'borderTopColor', Theme[`${theme}_chart_mountain_border_closed`]);
             this.stxx.setStyle('stx_mountain_chart', 'backgroundColor', Theme[`${theme}_chart_mountain_bg`]);
             this.stxx.setStyle('stx_mountain_chart', 'color', Theme[`${theme}_chart_mountain_bg_shade`]);
 
             // line chart
-            this.stxx.setStyle('stx_line_chart', 'color', closedChartColor);
-            this.stxx.setStyle('stx_line_up', 'color', closedChartColor);
-            this.stxx.setStyle('stx_line_down', 'color', closedChartColor);
-            this.stxx.setStyle('stx_line_even', 'color', closedChartColor);
+            this.stxx.setStyle('stx_line_chart', 'color', Theme[`${theme}_chart_default_border_closed`]);
+            this.stxx.setStyle('stx_line_up', 'color', Theme[`${theme}_chart_default_border_closed`]);
+            this.stxx.setStyle('stx_line_down', 'color', Theme[`${theme}_chart_default_border_closed`]);
+            this.stxx.setStyle('stx_line_even', 'color', Theme[`${theme}_chart_default_border_closed`]);
             // bar chart
-            this.stxx.setStyle('stx_bar_up', 'color', closedChartColor);
-            this.stxx.setStyle('stx_bar_down', 'color', closedChartColor);
-            this.stxx.setStyle('stx_bar_even', 'color', closedChartColor);
+            this.stxx.setStyle('stx_bar_up', 'color', Theme.chart_rise_up);
+            this.stxx.setStyle('stx_bar_down', 'color', Theme.chart_fall_down);
+            this.stxx.setStyle('stx_bar_even', 'color', Theme[`${theme}_chart_default_border_closed`]);
```

Comment:
not sure about this also, would u please check to see if we have this types or not